final-project/application.py

```diff
@@ -486,6 +486,84 @@ def search_category():
         users_selected_category=users_selected_category)
 
 
+# Store references to product research
+@app.route("/reference")
+def reference():
+    """Search for references to product research."""
+
+    # Keep the user logged in
+    user_id = session.get("user_id")
+
+    # Get the product name requested by the users
+    product_to_provide_references_for = request.args.get(
```

Comment:
not sure why you split this into 2 lines. Did it go over 80 chars? This looks less readable on Github.

Reply:
Yes, it was 82 characters or something. Where do you draw the line with number characters? Gut feeling about what is more or less readable?